waartaa/client/app/containers/chatapp/chatmenu.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Drawer from 'material-ui/Drawer';
+import {List, ListItem} from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+
+
+var ChatMenu = React.createClass({
+	render: function() {
+		return <Drawer open={true} zDepth={1}>
```

Comment:
Yeah, my sublime setup shows 4 space for a tab. So, I follow that way. I didn't realize it became 8 space here.